cmd/nexd/main.go

```diff
@@ -171,89 +244,95 @@ func main() {
 				Usage:    "Base64 encoded public `key` for the local host - agent generates keys by default",
 				EnvVars:  []string{"NEXD_PUB_KEY"},
 				Required: false,
+				Category: "Wireguard Options",
```

Comment:
perhaps add this at the top of the file to keep the string in one spot since it's repeated several times.

```go
const (
    wireguardOptions = "Wireguard Options"
)
```